src/current/_includes/releases/v24.3/v24.3.0-beta.2.md

```diff
@@ -0,0 +1,38 @@
+## v24.3.0-beta.2
+
+Release Date: October 28, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-3-0-beta-2-sql-language-changes">SQL language changes</h3>
+
+- If a table is the destination of a logical replication stream, then only schema change statements that are deemed safe are allowed on the table. Safe statements are those that do not result in a rebuild of the primary index and do not create an index on a virtual computed column. [#133266][#133266]
```

Comment:
Maybe link to https://www.cockroachlabs.com/docs/v24.2/indexes and https://www.cockroachlabs.com/docs/v24.2/computed-columns.html